lib/ansible/arguments/optparse_helpers.py

```diff
@@ -0,0 +1,377 @@
+# Copyright: (c) 2018, Ansible Project
```

Comment:
<nod> Yeah.  I thought about that with just the file at first but the current bin/ansible code made me decide against it.  I think the directory will work because I think the bin/ansible code is limited to files inside of the cli/ dir.  If that's so I'll move it.

Reply:
Moved in latest version